Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -194,6 +198,7 @@ internal class MonthView @JvmOverloads constructor(
         field = value
         labelsViewModel.update(value.calendarLabels)
         isRtl = value.isRtl
+        requestLayout()
```

Comment:
Since the row can change the height now, entire view height might be changed. By calling this method we make sure the size will be correctly re-calculated.